src/services/addAddresses.ts

```diff
@@ -0,0 +1,52 @@
+import { Customer } from '../types/user/customer';
+import { FormData } from '../types/user/formData';
+
+export const addAddresses = async (data: FormData, customer: Customer, userToken: string) => {
```

Comment:
Думаю будет лучше если мы будем принимать объект как пропс, так мы точно не ошибемся при вызове функции)
Можно добавить тип:
```suggestion
type address = { 
data: FormData; 
customer: Customer; 
userToken: string;
}
```

И можно сразу деструктурировать 
```suggestion
export const addAddresses = async ({data, customer, userToken}) => {
```

Reply:
Изменил. Думаю так лучше, не надо следит за очередностью пераметров.